src/coreclr/jit/compiler.cpp

```diff
@@ -4940,6 +4940,17 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     }
 #endif
 
+#ifdef DEBUG
+    // If we are going to simulate generating wasm control flow,
+    // transform any strongly connected components into reducible flow.
+    //
+    if (JitConfig.JitWasmControlFlow() > 0)
+    {
+        DoPhase(this, PHASE_DFS_BLOCKS_WASM, &Compiler::fgDfsBlocksAndRemove);
+        DoPhase(this, PHASE_WASM_TRANSFORM_SCCS, &Compiler::fgWasmTransformSccs);
+    }
+#endif
```

Comment:
Is it intentional this was placed before `PHASE_ASYNC`, even though it is eventually going to have to be after?

Reply:
Fixed in #121973